openmetadata-ui/src/main/resources/ui/src/components/Task/TaskTab/TaskTab.component.tsx

```diff
@@ -241,10 +241,7 @@ export const TaskTab = ({
       .then(() => {
         showSuccessToast(t('server.task-resolved-successfully'));
         rest.onAfterClose?.();
-
-        if (isTaskGlossaryApproval) {
```

Comment:
Can you verify the glossary approval task.

Reply:
yes, it working perfectly.